client-app/src/Components/Navbar.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import '../css/NavbarDropdown.css';
+
+
+const Navbar: React.FC = () => {
+    const [user, setUser] = useState<string>('');
+    const location = useLocation();
+    const [isLoggedIn, setIsLoggedIn] = useState<boolean>(false);
+
+    useEffect(() => {
+        const token = localStorage.getItem('token');
+        const name = localStorage.getItem('name');
```

Comment:
Remove the isLogged item from localStorage and rely solely on token to determine the logged-in state.

Reply:
I briefly tried to implement this, however it did not allow me to do signed in functionality when I relied only on token. I will try again at a later time